frontend/src/App.tsx

```
@@ -0,0 +1,26 @@
+import NotFound from './components/NotFound'
+import SignIn from './components/SignIn'
+import SignUp from './components/SignUp'
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'
+import client from './apolloClient'
+
+function App() {
+  return (
+    <ApolloProvider client={client}>
+      <Router>
+        <Routes>
+          <Route path='/signin' element={<GuestRoute children={<SignIn />} />} />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)